bundles/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/owserver/OwserverConnection.java

```diff
@@ -378,19 +425,29 @@ private void checkConnection() {
 
     /**
      * close the connection to the owserver instance after an error occured
+     * if {@link #CONNECTION_MAX_RETRY} is exceeded, the {@link #owserverConnectionState} is set to FAILED
+     * and reported to the {@link #thingHandlerCallback}.
```

Comment:
```suggestion
     * after {@link #CONNECTION_MAX_RETRY} is exceeded, connection state is reported to callback
```

Reply:
Should we leave the info in the comment that the new state is failed?

Reply:
Pushing a commit with ... hope that's ok...
if {@link #CONNECTION_MAX_RETRY} is exceeded, {@link #owserverConnectionState} is set to FAILED and state is reported to callback.